src/sectionproperties/analysis/fea.py

```diff
@@ -648,7 +696,9 @@ def element_stress(
 
         # extrapolate results to nodes, ignore numba warnings about performance
         with warnings.catch_warnings():
-            warnings.simplefilter("ignore", category=NumbaPerformanceWarning)
+            if USE_NUMBA:
+                warnings.simplefilter("ignore", category=NumbaPerformanceWarning)
```

Comment:
Change line 638 to:

```py
sig_zxy_mzz_gp = np.zeros((n_points, 2), order='F')        sig_zxy_vx_gp = np.zeros((n_points, 2), order='F')
sig_zxy_vy_gp = np.zeros((n_points, 2), order='F')
```

can suppress the warning so no need to import those extra utilities.